### ISSUE TITLE
get_daily_price_series seems to be having delay

### ISSUE THREAD

**Issue by suresh-n** [Shoonya-Dev/ShoonyaApi-py#90]
get_daily_price_series seems to be having so much delay , sometime its  return the output sometime its never ending which is causing the code to stuck and fail.

**Comment by rushey17** [comment]
+1

**Comment by pannet1** [comment]
this solution worked for me 

https://github.com/Shoonya-Dev/ShoonyaApi-py/issues/63#issuecomment-1296231695

**Comment by suresh-n** [comment]
@pannet1 yes its working on first try but when we hit second time it hung, do you see same ?

**Comment by rushey17** [comment]
Try yfinance as a alternative to get daily prices.

**Comment by mandu010** [comment]
I'm facing the same issue, and I've observed the issue only comes up on Monday.
@Shoonya-Dev

**Comment by sabyasm** [comment]
same here - can someone look into this please?

**Comment by Govardhanab3339** [comment]
Hi sir ,
i am a windows user and i am not getting ny response for NSE indexes Nifty 5o and Nifty Bank , kindly help , how to get i am using api=NorenApi()

**Comment by crackedminds333** [comment]
it's been a year and no update. i try to iterate over my watchlist and sometimes it just hangs waiting and hen returns a none type. sometimes it works. please fix this
